src/ts/components/dummycomponent.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, ComponentConfig } from './component';
+
+/**
+ * A dummy component whose sole purpose is to expose the {@link #prefixCss} method to classes that do not extend {@link Component}.
+ */
+export class DummyComponent extends Component<ComponentConfig> {
+    private static _instance: DummyComponent;
+
+    public static instance(): DummyComponent {
+        if (!DummyComponent._instance) {
+            DummyComponent._instance = new DummyComponent();
+        }
+
+        return DummyComponent._instance;
+    }
+
+    public prefixCss(cssClassOrId: string): string {
+        return super.prefixCss(cssClassOrId);
+    }
+}
```

Comment:
I would prefer to just export a function `prefixCSS` instead of a `DummyComponent`, that name is misleading. You can just export a function and hide the DummyComponent like 
```ts
class DummyComponent extends Component<ComponentConfig> {
    private static _instance: DummyComponent;

    public static instance(): DummyComponent {
        if (!DummyComponent._instance) {
            DummyComponent._instance = new DummyComponent();
        }

        return DummyComponent._instance;
    }

    public prefixCss(cssClassOrId: string): string {
        return super.prefixCss(cssClassOrId);
    }
}

/**
 * Prefixes any CSS Class or ID with the configured prefix.
 */
export function prefixCss(cssClassOrId: string): string {
    return DummyComponent.instance().prefixCss(cssClassOrId);
}
```
then using it also becomes easier to read `this.localStorageKey = prefixCss('subtitlesettings');`